---
layout: page
---
<h1 class="projects">    Projects</h1>

<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 1 illustration" src="../images/projectDL.png"/ Align="Left">
    </div>
    <div class="col-md-8">
        <h3 class="heading">Deep learning models for understanding the non-coding genome</h3>
        <div class="content">
            We are developing deep learning models to predict the impact of mutations in the DNA on cellular function across diverse cell types. We are also building interpretable models for understanding regulatory motifs that are predictive of cell-type specific activity of non-coding sequences.
            
            <br>
            People: Chendi Wang, Sasha Maslova
            <br>
            Alumni: Mark Ma
         
        </div>
    </div>
</div>

<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 1 illustration" src="../images/project1.png"/ Align="Left">
    </div>
    <div class="col-md-8">
        <h3 class="heading">Modeling the impact of genetic variation on multiple types of cellular traits</h3>
        <div class="content">
            We are developing integrative computational approaches for predicting the impact of genetic variation 
            on multiple types of cellular traits, including DNA methylation, histone modification,and gene expression.
            
            <br>
            People: Bernard Ng, Farnush Kiadeh
            <br>
            Alumni: Rosemary McCloskey, James Topham
            <br>
            Relevant publications: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24092820">here</a> and 
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/24786080">here</a>
        </div>
    </div>
</div>

<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 2 illustration" src="../images/deconv.png"/>
    </div>
    <div class="col-md-8">
        <h3 class="heading">Deconvolution of hidden and known confounding factors</h3>
        <div class="content">
            We are developing machine learning methods for modeling the effect of hidden and "partially known" confounding factors in gene expression and methylation data. 
            In this direction, we are currently developing methods for estimating cell type proportions in "genomics" samples from varied tissues including brain and cord blood.
            
            <br>
            Peaple: Louie Dinh
            <br>
            Relevant publications: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23874524">here</a>
        </div>
    </div>
</div>
<div class="section row">
    <div class="col-md-4 image">
        <img class="img-responsive" alt="Project 3 illustration" src="../images/mkm.png"/>
    </div>
    <div class="col-md-8">
        <h3 class="heading">Multi omics analysis of complex disease</h3>
        <div class="content">
            Working with clinical colleagues at UBC and elsewhere, we are developing association analysis approaches using multiple types of omics data. 
            Specifically, we are currently focusing on integrative analysis of genotype, gene expression, DNA methylation, and histone modification
            data in order to identify genes and networks that are associated with neurodegenerative diseases (including AD & PD).
            <br>
            People: Bernard Ng, Halldor Thorhallsson
        </div>
    </div>
</div>
<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 4 illustration" src="../images/hierarchy.png"/>
    </div>
    <div class="col-md-8">
        <h3 class="heading">Constructing tissue- and cell-type specific regulatory networks</h3>
        <div class="content">
            We have developed machine learning approaches for constructing gene functional association and gene regulatory networks in a context- and tissue-specific manner. 
            Expanding in this direction, we are interested in developing network construction algorithms that combine varied types of genomics data 
            (in particular data from DNA accessibility assays such as ATAC-Seq, gene expression data, and protein expression data) to model gene regulatory networks in a cell type specific manner.
            <br>
            Alumni: Emma Pierson
            <br>
            Relevant publications: <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=PMID%3A+25970446">here</a>,
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23284828">here</a>, and <a href="http://www.ncbi.nlm.nih.gov/pubmed/21877293">here</a>
        </div>
    </div>
</div>
